schema/test/unit/util_functions/get_fiscal_year_from_timestamp_test.sql

```diff
@@ -0,0 +1,19 @@
+begin;
+select plan(12);
+
+select is((select cif.get_fiscal_year_from_timestamp('2000-01-01 16:21:42.693489-07')), '1999/2000');
+select is((select cif.get_fiscal_year_from_timestamp('2000-02-01 16:21:42.693489-07')), '1999/2000');
+select is((select cif.get_fiscal_year_from_timestamp('2000-03-01 16:21:42.693489-07')), '1999/2000');
+select is((select cif.get_fiscal_year_from_timestamp('2000-04-01 16:21:42.693489-07')), '2000/2001');
```

Comment:
maybe we can test the edge condition with "March 31 at 23:59:59" and "Apr 1st at 00:00:00"